### ISSUE TITLE
sd_vvp deprecation warning in integrate_profile

### ISSUE THREAD

**Issue by peterdesmet** [adokter/bioRad#105]
```bash
> integrate_profile(example_vp)

Warning messages:
1: 'sd_vvp' is deprecated.
Use 'rvsd' instead.
See help("Deprecated") 
2: 'sd_vvp' is deprecated.
Use 'rvsd' instead.
See help("Deprecated") 
3: 'sd_vvp' is deprecated.
Use 'rvsd' instead.
See help("Deprecated") 
4: 'sd_vvp' is deprecated.
Use 'rvsd' instead.
See help("Deprecated") 
5: 'sd_vvp' is deprecated.
Use 'rvsd' instead.
See help("Deprecated") 
6: 'sd_vvp' is deprecated.
Use 'rvsd' instead.
See help("Deprecated")
```

Figure out why this (new) function is showing a `sd_vvp` deprecation warning.

**Comment by peterdesmet** [comment]
The same happens for the - granted, deprecated - function:

```R
> mtr(example_vp)
```

**Comment by adokter** [comment]
fixed by commit d6b4779af9ebd0edacdaa2c90a7bf6fb5b4320c4
